rosidl_generator_cpp/cmake/rosidl_generator_cpp_generate_interfaces.cmake

```diff
@@ -74,20 +78,34 @@ rosidl_write_generator_arguments(
   TARGET_DEPENDENCIES ${target_dependencies}
 )
 
+set(generator_module_paths "")
```

Comment:
This is fixed as of 94828cf, where it is used instead of `generator_module_path` in the `PYTHONPATH`.